SOURCES/cve-2019-1547.patch

```diff
@@ -0,0 +1,225 @@
+From 577f4211f1682110806c79b00cface4e6b73d4a1 Mon Sep 17 00:00:00 2001
+From: Lucas RAVAGNIER <lucas.ravagnier@vates.tech>
+Date: Tue, 29 Oct 2024 10:26:18 +0100
+Subject: [PATCH 1/3] [crypto/ec] for ECC parameters with NULL or zero
+ cofactor, compute it
+
+Backport notes: Droped CHANGES modifications
```

Comment:
I don't understand the comment (and s/Droped/Dropped).

Reply:
Reworded it for more clarity